itest/lnd_quiescence_test.go

```diff
@@ -0,0 +1,32 @@
+package itest
+
+import (
+	"github.com/btcsuite/btcd/btcutil"
+	"github.com/lightningnetwork/lnd/lnrpc"
+	"github.com/lightningnetwork/lnd/lntest"
+	"github.com/stretchr/testify/require"
+)
+
+// testQuiescence tests whether we can come to agreement on quiescence of a
+// channel. We initiate quiescence via RPC and if it succeeds we verify that
+// the expected initiator is the resulting initiator.
+//
+// NOTE FOR REVIEW: this could be improved by blasting the channel with HTLC
```

Comment:
yeah it'd be nice to test more scenarios, tho I really like how simple this test is.